optaweb-vehicle-routing-frontend/.env

```diff
@@ -0,0 +1,2 @@
+# https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables
+REACT_APP_BACKEND_URL=.
```

Comment:
I think using an empty string will also work

Reply:
Thanks, I'll give it a try.